dnf-json

```diff
@@ -0,0 +1,200 @@
+#!/usr/bin/python3
```

Comment:
This is now used only to populate the distro files, right? In which case: we should probably document somewhere how to re-run that?

Reply:
Made https://github.com/osbuild/image-builder/issues/126 to track that